lib/remarkMdxLayout.mjs

```diff
@@ -30,14 +30,15 @@ export const remarkMdxLayout = () => {
   
           const outline = ${Serializer.run(outline)}
   
-          export const getStaticProps = async () => {
+          export const getStaticProps = async (params) => {
             const locale = '${locale}'
             const navItems = await getNavItems(locale)
 
             return {
               props: {
                 locale,
                 navItems,
+                ...(typeof mdxGetStaticProps !== 'undefined' && (await mdxGetStaticProps(params)).props)
```

Comment:
since we can't have 2 `getStaticProps` on the page I made some workaround 
<img width="387" alt="image" src="https://user-images.githubusercontent.com/7361780/215564301-a587e907-ac86-4b1f-9ce8-4f8f579ede03.png">